tests/integration/api_v2/events/blocks.test.js

```diff
@@ -0,0 +1,47 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const config = require('../../../config');
+
+const {
+	subscribeAndReturn,
+	closeAllConnections,
+} = require('../../../helpers/socketIoSubscribe');
+
+const {
+	goodRequestSchema,
+	metaSchema,
+} = require('../../../schemas/httpGenerics.schema');
+
+const {
+	blockSchemaVersion5,
+} = require('../../../schemas/block.schema');
+
+const endpoint = `${config.SERVICE_ENDPOINT_RPC}/blockchain`;
+
+describe('Test subscribe API latest block event', () => {
+	it('event update.block', async () => {
+		const response = await subscribeAndReturn(endpoint, 'update.block');
```

Comment:
What is the timeout for the event to occur? Is it limited only by Jenkins?

Reply:
Yes, timeout is 15 seconds which is limited by `Jenkins.